crud.py

```diff
@@ -0,0 +1,66 @@
+from sqlalchemy.orm import Session
+import models
+import schemas
+
+
+def get_all_authors(db: Session) -> list[str]:
+    return db.query(models.Author).all()
+
+
+def get_author_by_name(db: Session, name: str) -> str or None:
```

Comment:
```suggestion
def get_author_by_name(db: Session, name: str) -> Optional[models.Author]:
```
Please fix in
.first return object that specified inside .query and also can return None